File: Java/Car.java
class Car {
  Integer id;
  String license;
  Account driver;
  private Integer passenger;

  public Car(String license, Account driver) {
    this.license = license;
    this.driver = driver;
  }

  void print_data() {
    System.out.println("License: " + license + ", Driver name: " + driver.name);
  }

  public Integer get_passenger() {
    return  passenger;
  }

  public void set_passenger(Integer passenger) {
    if (passenger == 4) {
      this.passenger = passenger;
      System.out.println("Pasajeros asignados");
    } else {
      System.out.println("Necesitas asignar 4 pasajeros");
    }
  }
}

File: PHP/Car.php
<?php

class Car {
  public $id;
  public $license;
  public $driver;
  public $passengers;

  public function __construct($license, $driver) {
    $this->license = $license;
    $this->driver = $driver;
  }

  public function print_data_car() {
    print "License: $this->license, Name: $this->driver->name, Document: $this->driver->document";
  }
}

?>
